MQTT-C_Client/README.md

```diff
@@ -0,0 +1,82 @@
+# MQTT project
+
+The goal of this project is to show how to add MQTT support to an Azure Sphere project, in this case using the [Eclipse Mosquitto MQTT Broker](https://mosquitto.org/) and a PC-based Python application.
```

Comment:
I think you could call out the role of each part of the solution here - it wasn't clear to me at first glance what each part was doing and how they interace.